app/models/answer.rb

```diff
@@ -7,4 +7,12 @@ class Answer < ApplicationRecord
   def find_user
     user.name unless user.nil?
   end
+
+  def self.populate_answer(comment_params)
+    if comment_params.has_key?(:answer)
+      Answer.find(comment_params[:answer])
```

Comment:
A better syntax for this might be "if comment_params[:answer]" that will check the existence without having to iterate through comment_params.